.gitignore

```diff
@@ -106,3 +106,6 @@ venv.bak/
 # editors
 .idea/
 .vscode/
+
+# Seed json
+results.json
```

Comment:
let's just check it in for now. that way all developers can get easy access to it.